pom.xml

```diff
@@ -229,6 +229,7 @@
         <testFtpServerVersion>1.0.6</testFtpServerVersion>
         <tomcatVersion>6.0.44</tomcatVersion>
         <unirestVersion>1.4.9</unirestVersion>
+        <weaveVersion>${project.version}</weaveVersion>
```

Comment:
For now the version matches ours but yes, perhaps we should use 4.0-SNAPSHOT since we are most likely going to diverge soon.